client/src/extension.ts

```diff
@@ -30,6 +30,13 @@ var terminal;
 const fs = require('fs');
 const url = require('url');
 
+const FILE_ASSOCIATIONS : string = 'files.associations';
+const REACH_FILE_SELECTOR : string = '*.rsh';
+const JAVASCRIPT : string = 'javascript';
+const WORKSPACE_SETTINGS_PATH : string = `${rootFolder}${path.sep}.vscode/settings.json`;
+const CANNOT_CREATE_SETTINGS : string = `Could not create .vscode/settings.json:`; 
```

Comment:
used once: inline

Reply:
This has been updated and is pending another review